api/v1alpha1/openapi.yaml

```diff
@@ -4989,3 +5004,35 @@ components:
       required:
         - conditions
       type: object
+    Version:
+      properties:
```

Comment:
Let's make all properties `required`, so we don't have to deal with pointers.

Reply:
I agree to make everything required, but wondering if 10 properties is overkill for describing a version?

Reply:
CLI version got 10 fields. This way we know where exactly it came from,  so I don't see why not to have the same fields for API. Especially as all of them are already here

Reply:
Is that a potential security risk? Do we need a way to disable this info or put it behind RBAC?

Reply:
It should definitely be behind RBAC.

Again, why isn't one string enough?  For example 0.4.0-rc2 if it's an official release, or a git version otherwise?

Reply:
done